cmd/tendermint/commands/testnet.go

```diff
@@ -134,32 +134,14 @@ func testnetFiles(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	// Gather persistent peer addresses.
-	var (
-		persistentPeers string
-		err             error
-	)
 	if populatePersistentPeers {
-		persistentPeers, err = persistentPeersString(config)
+		err := populatePersistentPeersInConfigAndWriteIt(config)
```

Comment:
why do we only write the config file if populatePersistentPeers?

Reply:
ah - I think this accidentally undid the fix from https://github.com/tendermint/tendermint/pull/2983/files